src/components/molecules/NavSearchBar/NavSearchBar.tsx

```diff
@@ -79,72 +82,78 @@ const NavSearchBar = () => {
     <div className="nav-search-links">
       <div className="nav-search-icon" />
       <NavSearchBarInput value={searchQuery} onChange={setSearchQuery} />
+
       {isTruthy(searchQuery) && (
         <div className="nav-search-close-icon" onClick={clearSearchQuery} />
       )}
-      {isTruthy(numberOfSearchResults) && (
-        <>
```

Comment:
Removed the unnecessary fragment, as we are already within a single `div` anyway.

(Reviewing this with 'show whitespace changes' turned off will minimize the changes you need to review)